BeeSwift/Gallery/GalleryViewController.swift

```diff
@@ -482,18 +483,20 @@ class GalleryViewController: UIViewController, UICollectionViewDelegateFlowLayou
     }
     
     private func configureDataSource() {
-        let cellRegistration = UICollectionView.CellRegistration<GoalCollectionViewCell, Goal> { cell, indexPath, goal in
+        let cellRegistration = UICollectionView.CellRegistration<GoalCollectionViewCell, NSManagedObjectID> { [weak self] cell, indexPath, goalObjectId in
+            let goal = self?.fetchedResultsController.fetchedObjects?.first(where: { $0.objectID == goalObjectId })
```

Comment:
Why this over `fetchedResultsController.object(at: indexPath)` ?

Reply:
no good reason. changed